packages/itwinui-css/src/table/table.scss

```diff
@@ -251,12 +253,15 @@
     &.iui-loading,
     .iui-table-cell[aria-disabled='true'],
     &.iui-table-expanded-content[aria-disabled='true'] {
+      --_iui-table-cell-icon-fill: var(--iui-color-foreground-4);
+      --_iui-table-cell-image-filter: grayscale(100%);
```

Comment:
This applies to icons, images, and `iui-avatar` so I'm not sure what the variable should be named.

Reply:
Naming looks good to me.